builtin_interfaces/Time stamp
int32 id
float64 position
float64 speed
float64 qpps
